kod/object/active/holder/nomoveon/battler/player.kod

```diff
@@ -5501,13 +5501,17 @@ messages:
             if IsClass(oJalaSpell,&Restorate)
             {
                iJalaPower = Nth(Nth(lJalaInfo,3),3);
+
+               iTime = bound(iTime,1000,25000);
                iTime = Send(oJalaSpell,@AdjustHealthTime,#time=iTime,
                             #iSpellPower=iJalaPower);
+
+               return bound(iTime,670,25000);
             }
          }
       }
 
-      return bound(iTime,1000,60000);
+      return bound(iTime,1000,25000);
```

Comment:
I see your comment about changing this line, but since it isn't related to Restorate, I'd like to see it reverted, and you can make a separate PR to discuss it if you want.

Reply:
Ok.  Upper limits for both with and without the Jala spell are returned to 60 seconds.  We can discuss fixing this in a separate PR.